hack/e2e/kops/kops.sh

```diff
@@ -119,15 +116,15 @@ function kops_patch_cluster_file() {
   if [ -n "$ROLE" ]; then
     FILTER="$FILTER | select(.spec.role==\"$ROLE\")"
   fi
-  jq "$FILTER" "$CLUSTER_FILE_JSON" > "$CLUSTER_FILE_0"
+  jq "$FILTER" "$CLUSTER_FILE_JSON" >"$CLUSTER_FILE_0"
 
   # Patch only the json objects
-  kubectl patch -f "$CLUSTER_FILE_0" --local --type merge --patch "$(cat "$KOPS_PATCH_FILE")" -o json > "$CLUSTER_FILE_1"
```

Comment:
Nit: Can you put whatever linter you used for our scripts into the makefile documentation? In a future PR we can add an update target that runs the bash linter.

Reply:
It's already in this PR, see target `update/shfmt`